code/modules/power/supermatter/supermatter.dm

```diff
@@ -240,6 +254,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		/datum/gas/healium = 1,
 		/datum/gas/proto_nitrate = 1,
 		/datum/gas/zauker = 1,
+		/datum/gas/miasma = 0,
```

Comment:
Setting it to 0 and leaving it out of the equation have the same effect, if you want it to be a positive impact set it to 1

Reply:
I don't really want miasma to have a high gasmix power. The point of miasma is the matter power interaction. Besides, at the very least it's more potent than nitrogen's -1.

Reply:
Alright, but this quite literally does nothing 
![image](https://user-images.githubusercontent.com/58055496/108462208-11a63000-7231-11eb-94d1-59baa906dd84.png)
total +=  ratio * 0 will always be 0

Reply:
last I checked null counts at 1 for multiplication.

Reply:
Alright. I boosted it a bit.